pkg/controller/utils/component.go

```diff
@@ -663,6 +669,47 @@ func setImagePullPolicy(podSpec *v1.PodSpec) {
 	}
 }
 
+// ensureTLSCiphers sets the TLSCipherSuites configuration as a Env Var to the Deployments and DaemonSets.
+func ensureTLSCiphers(obj client.Object, c client.Client) error {
+	var containers []v1.Container
+	switch obj := obj.(type) {
+	case *apps.Deployment:
+		containers = obj.Spec.Template.Spec.Containers
+	case *apps.DaemonSet:
+		containers = obj.Spec.Template.Spec.Containers
+	default:
+		return nil
+	}
+
+	_, installationSpec, err := GetInstallation(context.Background(), c)
```

Comment:
Could you please pass the context in from the calling createOrUpdateObject function and use that instead.